platforms/hdp2.6.3/docs/HDP-installation.md

```diff
@@ -148,6 +153,10 @@ the admin plugin for Splice Machine is added to the Ranger web application.
 
 The service can be installed by executing the following from a command line on the machine where the Ambari Service resides.
 
+You can download the file `ranger-servicedef-splicemachine.json` at [here](https://raw.githubusercontent.com/splicemachine/spliceengine/branch-2.5/assembly/hdp2.6.3/src/main/resources/stacks/HDP/2.6/services/SPLICEMACHINE/configuration/ranger-servicedef-splicemachine.json).
```

Comment:
Shouldn't this be included on the Splice distribution? Otherwise there could be differences between the deployed Splice version and the Ranger json